datumaro/components/media.py

```diff
@@ -335,14 +501,23 @@ def save(self, path, crypter: Crypter = NULL_CRYPTER):
             save_image(path, self.data)
 
 
-class VideoFrame(Image):
+class VideoFrame(ImageFromNumpy):
```

Comment:
Is this okay for getting data or size from `ImageFromNumpy`?

Reply:
Yes, `VideoFrame` is just a thin wrapper around `Image` so `data` would be used as usual and `size` is overwrited.